App.js

```diff
@@ -179,9 +179,25 @@ export default class App extends Component<Props> {
       });
   };
 
-  updateAuthUser(newUser) {
-    this.setState({ auth: newUser });
-  }
+  toggleStarred = async (id, starDict) => {
+    let curItems;
+    if (this.state.starredItems == null) {
```

Comment:
when would this ever be the case?

Reply:
when asyncstorage first gets initialized (aka when somebody opens the app for the first time)